_lessons/python-scatter/01-profiling.md

```diff
@@ -33,23 +33,21 @@ cd original
 
 ## Profiling Python code with *cProfile*
 
-The *cProfile* profiler is one implementation of the Python profiling interface. It measures the time spent within a function and the amount of calls.
+The *cProfile* profiler is one implementation of the Python profiling interface. It measures the time spent within a function and the number of times 
+the function is called.
 
-**Note:** The timing information should not be taken as absolute values, since the profiling itself could possibly extend the run time, in some cases.
-
-Run the following command to profile the code:
+**Note:** The timing information should not be taken as absolute values, since the profiling itself could extend the run time.
 
+Replace `python scatter.py` with 
```

Comment:
I would more prefer to write: 
The command line options `-m cProfile -o output.pstats` call the *cProfile* library and writes output the profiling data into "output.pstat". Thus in the present case we call:

Reply:
Thanks. Added:
"Options `-m cProfile -o output.pstats` call the cProfile library and save the profiling data in file *output.pstat*."